.github/workflows/ci.yml

```diff
@@ -445,6 +437,122 @@ jobs:
           TEST_RUNNER_EXTRA: ${{ github.event_name != 'pull_request' && '--extended' || '' }}
         run: py -3 test/functional/test_runner.py --jobs $NUMBER_OF_PROCESSORS --ci --quiet --tmpdirprefix="$RUNNER_TEMP" --combinedlogslen=99999999 --timeout-factor=$TEST_RUNNER_TIMEOUT_FACTOR $EXCLUDE $TEST_RUNNER_EXTRA
 
+  windows-ucrt-cross:
```

Comment:
Is there a way to do this without copy-pasting/duplicating > 100 lines of CI config (where both copies need to be kept in sync).